src/lib.rs

```diff
@@ -247,7 +247,7 @@ pub use self::result::Result::{Err, Ok};
 pub mod prelude {
     pub use crate::error::ErrorStack;
     pub use crate::result::Result;
-    pub use crate::result::Result::{Err, Ok};
+    pub use crate::result::Result::Ok;
```

Comment:
My intuition is that it would be best to avoid shadowing the std `Ok` and `Err` constructors, though I think this is more important for `Err` than `Ok` because it allows you to make the `ErrorStack` construction implicit as part of the conversion from `?`

Reply:
Seems like if we shadow one then we should shadow both, or shadow neither.